src/srcom/slash_commands.ts

```diff
@@ -384,7 +393,8 @@ async function sendCommand(
 				embeds: [
 					new Embed({
 						description:
-							`Unexpected Error, please report this to a developer: ${command}\n\`/${i.data.name} ${
+							`Unexpected Error, please report this to a [developer](https://github.com/AnInternetTroll/speedrunbot-slash):
```

Comment:
Can this link to https://github.com/AnInternetTroll/speedrunbot-slash/issues/new